packages/flutter/lib/src/material/button_theme.dart

```diff
@@ -270,13 +272,14 @@ class ButtonThemeData extends Diagnosticable {
     Color hoverColor,
     Color highlightColor,
     Color splashColor,
-    this.colorScheme,
+    this.colorScheme = const ColorScheme.light(),
```

Comment:
You might want to add in the constructor's documentation that `colorScheme` must not be null

Reply:
In the constructors documentation, it is already mentioned that `colorScheme` cannot be null.
https://github.com/flutter/flutter/blob/bb8b22e95330b4bfddb6a13308eb0b2615535dd5/packages/flutter/lib/src/material/button_theme.dart#L66-L71